src/proto/workload.proto

```diff
@@ -0,0 +1,152 @@
+syntax = "proto3";
```

Comment:
Maybe instead of having a copy of the Workload API we could have `spiffe/spiffe` repo as a submodule and use `spiffe/main/standards/workloadapi.proto` directly!

Reply:
Not so sure about this, I'm not a big fan of `git submodules` 😁  I'd leave as it is and put more thinking on this later on.